runtime/storage-proxy.js

```diff
@@ -189,12 +191,12 @@ class StorageProxyBase {
     }
   }
 
-  generateID(component) {
-    return this._pec.generateID(component);
+  generateID() {
+    return `${this._baseForNewID}:${this._localIDComponent++}`;
   }
 
   generateIDComponents() {
-    return this._pec.generateIDComponents();
+    return {base: this._baseForNewID, component: () => this._localIDComponent++};
```

Comment:
I think this means you can remove the _pec field from StorageProxy.

Reply:
done